sites/cheerpx/src/content/docs/13-tutorials/setup-svelte.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Svelte setup
+description: Setting up Svelte app
+---
+
+This tutorial will explain how to create a application with Svelte that works together with CheerpX.
+
+## Install Svelte
+
+First create a new Svelte app by running:
+
+```
+npm create svelte@latest my-app
+```
+
+1. Select the Skeleton Project option.
+2. Enable Typescript if needed.
+3. There are no additional options required for this tutorial.
+
+Next, navigate to your project directory and install the dependencies:
+
+```
+cd my-app
+npm install
+npm run dev
+```
+
+At this point, the development server will run from another terminal in the background.
+
+We're able to test the development server by visiting: http://localhost:5173/
+
+## Install Cheerpx
+
+Next, let's install CheerpX.
+
+This [link] will direct you to the installation page of CheerpX.
+
+For this to work all top level imports used by the CheerpX NPM module need to be enabled.
+
+We'll also need to enable Cross origin isolation. It's required since CheerpX uses SharedArrayBuffer.
+
+Add the following code in vite.config.ts:
+
+```
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vite';
+
+const viteServerConfig = () => ({
+    name: 'add-headers',
+    configureServer: (server) => {
+        server.middlewares.use((req, res, next) => {
+            res.setHeader("Cross-Origin-Opener-Policy", "same-origin");
+            res.setHeader("Cross-Origin-Embedder-Policy", "require-corp");
+            next();
+        });
+    }
+});
+
+export default defineConfig({
+  optimizeDeps:{
+    esbuildOptions: {
+      target: "es2022",
+    }
+  },
+        plugins: [sveltekit(), viteServerConfig()]
+});
+```
+
+Add src/routes/+page.ts and disable [SSR]. CheerpX code needs to run on the side:
+
+```
+export const ssr = false;
+```
+
+## Script
+
+Add the following code to src/routes/+page.svelte to run bash using CheerpX. for a more detailed explanation click the following [instructions].
```

Comment:
We will now modify ` src/routes/+page.svelte` to replace the Svelte example with a full-screen console running `bash` using CheerpX. For more details on the CheerpX logic itself, please see our [Getting Started] guide.